TeamCode/src/main/java/org/firstinspires/ftc/teamcode/Teleop/IntoTheDeepTeleOp.java

```diff
@@ -52,6 +68,13 @@ public void init() {
 
         armLeftFront = hardwareMap.get(Servo.class, "servo1");
         armRightFront = hardwareMap.get(Servo.class, "servo2");
+        //get our analog input from the hardwareMap
+        armLeftFrontEncoder = hardwareMap.get(AnalogInput.class, "armLeftFrontEncoder");
+// get the voltage of our analog line
+// divide by 3.3 (the max voltage) to get a value between 0 and 1
+// multiply by 360 to convert it to 0 to 360 degrees
```

Comment:
we're not going to use axon encoders at this point so I don't think we need this anymore